x-pack/test/functional_endpoint/services/endpoint_policy.ts

```diff
@@ -34,87 +38,119 @@ export interface PolicyTestResourceInfo {
+      let agentConfig: CreateAgentConfigResponse['item'];
+      try {
+        const { body: createResponse }: { body: CreateAgentConfigResponse } = await supertest
+          .post(INGEST_API_AGENT_CONFIGS)
           .set('kbn-xsrf', 'xxx')
-          .send()
+          .send({ name: 'East Coast', description: 'East Coast call center', namespace: '' })
           .expect(200);
+        agentConfig = createResponse.item;
+      } catch (error) {
+        log.error(error);
+        throw new Error(`Unable to create Agent Config via Ingest!`);
       }
 
-      // create agent config
-      const {
-        body: { item: agentConfig },
-      }: { body: CreateAgentConfigResponse } = await supertest
-        .post(INGEST_API_AGENT_CONFIGS)
-        .set('kbn-xsrf', 'xxx')
-        .send({ name: 'East Coast', description: 'East Coast call center', namespace: '' })
-        .expect(200);
+      // Retrieve the Endpoint package information
+      let endpointPackageInfo: GetPackagesResponse['response'][0] | undefined;
+      try {
+        const { body: secPackages }: { body: GetPackagesResponse } = await secPackagesRequest;
+        endpointPackageInfo = secPackages.response.find(
+          epmPackage => epmPackage.name === 'endpoint'
+        );
+        if (!endpointPackageInfo) {
+          throw new Error(`Endpoint package was not found via ${SECURITY_PACKAGES_ROUTE}`);
+        }
+      } catch (error) {
+        log.error(error);
+        throw new Error(`Unable to retrieve Endpoint package via Ingest!`);
+      }
 
       // create datasource and associated it to agent config
-      const {
-        body: { item: datasource },
-      }: { body: CreateDatasourceResponse } = await supertest
-        .post(INGEST_API_DATASOURCES)
-        .set('kbn-xsrf', 'xxx')
-        .send({
-          name: 'Protect East Coast',
-          description: 'Protect the worlds data - but in the East Coast',
-          config_id: agentConfig.id,
-          enabled: true,
-          output_id: '',
-          inputs: [
-            // TODO: should we retrieve the latest Endpoint Package and build the input (policy) from that?
-            {
-              type: 'endpoint',
-              enabled: true,
-              streams: [],
-              config: {
-                policy: {
-                  value: policyConfigFactory(),
+      let datasource: CreateDatasourceResponse['item'];
+      try {
+        const { body: createResponse }: { body: CreateDatasourceResponse } = await supertest
+          .post(INGEST_API_DATASOURCES)
+          .set('kbn-xsrf', 'xxx')
+          .send({
+            name: 'Protect East Coast',
+            description: 'Protect the worlds data - but in the East Coast',
+            config_id: agentConfig.id,
+            enabled: true,
+            output_id: '',
+            inputs: [
+              // TODO: should we retrieve the Endpoint Package definition information (another API call) and build the input (policy) from that?
```

Comment:
thoughts on just removing this comment?

Reply:
Yeah, I will remove it. I do think that this should be implemented in the future, but will try to track it as part of the Issue we have to add the default policy to the Endpoint package.